ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/CommitContext.scala

```diff
@@ -44,8 +45,6 @@ private[kvutils] case class CommitContext(
   // pre-execution.
   var outOfTimeBoundsLogEntry: Option[DamlLogEntry] = None
 
-  def preExecute: Boolean = recordTime.isEmpty
```

Comment:
Pre-execution and the absence of a record time (because it is only determined at post-execution) are equivalent. Why are you separating them?